homeassistant/components/cookidoo/quality_scale.yaml

```
@@ -0,0 +1,90 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: No service actions implemented
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: No service actions implemented
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions:
+    status: exempt
+    comment: No special external action required
+  entity-event-setup:
+    status: exempt
+    comment: No callbacks are implemented
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  config-entry-unloading: done
+  log-when-unavailable:
+    status: done
+    comment: Offloaded to coordinator
+  entity-unavailable:
+    status: done
+    comment: Offloaded to coordinator
+  action-exceptions:
+    status: exempt
+    comment: No service actions implemented
+  reauthentication-flow: done
+  parallel-updates: done
+  test-coverage: done
+  integration-owner: done
+  docs-installation-parameters: done
+  docs-configuration-parameters:
+    status: exempt
+    comment: No options flow
+
+  # Gold
+  entity-translations: done
+  entity-device-class:
+    status: exempt
+    comment: currently no platform with device classes
+  devices: done
+  entity-category: done
+  entity-disabled-by-default:
+    status: exempt
+    comment: No disabled entities implemented
+  discovery:
+    status: exempt
+    comment: Nothing to discover
+  stale-devices:
+    status: exempt
+    comment: No stale entities possible
+  diagnostics: todo
+  exception-translations: done
+  icon-translations: done
+  reconfiguration-flow: done
+  dynamic-devices:
+    status: exempt
+    comment: No dynamic entities available
+  discovery-update-info:
+    status: exempt
+    comment: No discoverable entities implemented
+  repair-issues:
+    status: exempt
+    comment: No issues/repairs
+  docs-use-cases: todo
+  docs-supported-devices: todo
+  docs-supported-functions: todo
+  docs-data-update: done
```

```suggestion
  docs-data-update: todo
```
Update process/interval not clear to me from the docs

---

I added a section for custom polling intervals to clarify this. @zweckj what do you think?

---

that's probably more extensive than needed, but fine for me 👍

---

I did the same for other integration and just copied from there. (ex. `swiss_public_transport` and `bring`)